### DISCUSSION TITLE
Does TypeChat support browser?

### DISCUSSION THREAD

**Discussion by MRYingLEE** [microsoft/TypeChat#125] [Q&A]
I am trying TypeChat.
I notice all examples run in node.js.

Just want to confirm that TypeChat cannot run in browser.

Please clarify.

**Comment by DanielRosenwasser** [comment]
TypeChat can run in a browser - though you may have had issues with bundling it or finding a good workflow for including a schema at runtime.

It should be easier as of [TypeChat 0.1.0](https://microsoft.github.io/TypeChat/blog/announcing-typechat-0-1-0/) which has split `processRequests` into its own entry-point, and allows you to construct a validator out of Zod schemas. That means you can just create objects at runtime, and provide it to TypeChat.
